frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -215,7 +215,7 @@ const Sidebar = ({
                   <img draggable={false} src={coins} alt={t('sidebar.exchanges')}/>
                 </div>
                 <span className={style.sidebarLabel}>
-                  {hasOnlyBTCAccounts ? t('accountInfo.buyCTA.buy', { unit: 'Bitcoin' }) : t('sidebar.buy')}
+                  {hasOnlyBTCAccounts ? t('accountInfo.exchangeCTA.exchange', { unit: 'Bitcoin' }) : t('sidebar.exchange')}
```

Comment:
here unit is always 'Bitcoin' but in other places it can be 'crypto' but sidebar.exchange is always 'Exchange crypto'.

How about removing sidebar.exchange.

Also maybe consider moving the 'accountInfo.exchangeCTA.exchange' key to 'exchange.exchangeCTA' or similar, so that it is not under accountInfo?